### DISCUSSION TITLE
[Discussion] Public communication channel

### DISCUSSION THREAD

**Discussion by UnseenWizzard** [Dynatrace/dynatrace-configuration-as-code#204] [Archive] [answered]
Having open sourced our work, we want to open our communication channels as well. 
That means moving away from our internal Slack channels to something that external contributors [0] can use as well. 

Let's use this to discuss options! 

A few ideas: 
Slack - pro: extra workspace can just be opened in Slack apps we already use internally all the time, con: text only
Discord - pro: widely used, has voice chat, con: extra tool

Decision Deadline: 2020-12-12
Participation Required: @didiladi , @tatomir146 , @CruzanCaramele 

[0] e.g. @goberchtold (testing if reaching out this way works)

**Comment by ghost** [comment]
What can Slack do that Github can't?
Only thing I can thing of is **I** choose (as a contributor) which slack channel I ask my question in. The corresponding workflow in Git would be to raise an issue but then **you** (as an admin) need to add tags.

This could be fixed by using [Github issue templates](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates#issue-templates) which will automatically add labels to anyone's posts

- Question
- Bug
- Security
- etc.

**Comment by ghost** [comment]
Side note: I've never liked the wording of GitHub "issues". "Discussions" would be more appropriate and less dramatic IMO. Every time I see "10,323 issues" I always think "ooh that repo is buggy!"

**Comment by didiladi** [comment, marked as answer]
We could try out GitHub discussions within this repository: https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions.

Its still in Beta, but let's try it out to see if it works for us. What do you think?

![image](https://user-images.githubusercontent.com/1592770/101594189-8d922980-39f1-11eb-9486-a101da2f5abe.png)

**Comment by patrickpichler** [comment]
Wouldn't [gitter](https://gitter.im/) also be worth taking into consideration? 

Since it is using the matrix protocol you can join rooms via any matrix client.
